Gameplay/PlayerController.cpp

```diff
@@ -206,11 +206,11 @@ void Hachiko::Scripting::PlayerController::HandleInputAndStatus()
 
 	if (_is_god_mode && Input::IsKeyPressed(Input::KeyCode::KEY_Q))
 	{
-		_movement_direction += math::float3::unitY;
+		_player_position += math::float3::unitY * 0.5f;
```

Comment:
maybe it would be better to have a vertical speed variable instead of this magic number